libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx

```diff
@@ -47,7 +42,7 @@ export function StepGeneral(props: StepGeneralProps) {
     [cloudProviders]
   )
 
-  const buildRegions: () => Value[] = useCallback(
```

Comment:
why did you change from useCallback to useMemo ? (it's for my understanding of the differences)

Reply:
Here, useMemo it's the better choice because useCallback returns a memoized function and useMemo returns a memoized value